app/services/analytics.rb

```diff
@@ -129,6 +129,8 @@ def browser_attributes
   ADD_EMAIL_CONFIRMATION_RESEND = 'Add Email: Email Confirmation requested due to invalid token'.freeze
   ADD_EMAIL_VISIT = 'Add Email: enter email visited'.freeze
   AUTHENTICATION_CONFIRMATION = 'Authentication Confirmation'.freeze
+  AUTHENTICATION_CONFIRMATION_CONTINUE = 'Authentication Confirmation: Continue selected'.freeze
+  AUTHENTICATION_CONFIRMATION_RESET = 'Authentication Confirmation: Reset selected'.freeze
```

Comment:
Can we check that these events are logged somewhere in specs (e.g. feature specs for this flow)?

Reply:
Ah yeah, I wrote a controller spec with the idea to test these and never actually tested that they were logged

Reply:
Just pushed some checks for these events

Reply:
The other thing I was curious about is if we were logging anything previously that we care to be continuing to log in this new redirect flow.